### ISSUE TITLE
[QUESTION] - HTTP Error Response Handling

### ISSUE THREAD

**Issue by go4cas** [d0whc3r/vue-auth-plugin#64]
I would just like to check something regarding handling of HTTP error codes: when I pass invalid credentials to the `/login` endpoint of my API, it will respond with a `401` response.

In my front-end app, I have this code to attempt a user login:
```javascript
          this.$auth.login({
            email: this.loginForm.email,
            password: this.loginForm.password
          })
            .then((response) => {
              this.$Notice.success({
                title: `logged onto console`
              })
            })
            .catch((err) => {
              this.$Notice.error({
                title: `could not log on`,
                desc: err
              })
            })
```

I was expecting the notification in the `catch` block to fire on the `401`, but it goes into the `then` block.

Am I doing something wrong? Or, is this something that I need to configure on either the `vue-auth-plugin` config, or the global `axios` config?

**Comment by d0whc3r** [comment]
If $auth.login() was wrong you will see a `console.warn` something like `[vue-auth-plugin] Login error Request failed with status code 401`

**Comment by d0whc3r** [comment]
Ok I published a new version in npm (1.1.1) it solves this issue feel free to reopen if it is not solved.

Thank you very much

**Comment by go4cas** [comment]
I have upgraded, and the `401` now hits the `catch` block. Thanks for the very quick turn-around on this!
